applications/NXscan.nxdl.xml

```diff
@@ -27,6 +27,21 @@
 	    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
 	    >
+    <symbols>
+        <doc>
+		    The symbol(s) listed here will be used below to coordinate datasets
+			with the same shape.
+        </doc>
+		<symbol name="nP">
+		    <doc>Number of points</doc>
+        </symbol>
+		<symbol name="xDim">
+		    <doc>xDim description</doc>
```

Comment:
number of X values

Reply:
see https://github.com/nexusformat/definitions/pull/819#discussion_r510940391